bin/build

```diff
@@ -9,7 +9,7 @@ rm -rf ./.aws-sam \
        ./vendor/bundle
 
 # Build SAM
-sam build
+sam build --profile wustl --region us-east-2
```

Comment:
Question should the region be us-east-2 or a env variable as you have it in the bin/_deploy script for consistency sake?